app/javascript/angular/code/services/_fixed_sessions.js

```diff
@@ -66,43 +67,38 @@ export const fixedSessions = (
     },
 
     deselectSession: function() {
-      var session = sessionsUtils.selectedSession(this);
-      if (!session) return;
-      session.loaded = false;
```

Comment:
I don't see any place where it is useful. It was anly use in `drawSession.drawMobileSession()` but since this method is only called from a callback after a successful fetch request, then I think it's safe to assume that the session is ready to be drawn.